@xen-orchestra/web-core/lib/components/input/InputLabel.vue

```diff
@@ -0,0 +1,74 @@
+<!-- v2.0 -->
+<template>
+  <div :class="status" class="input-label">
+    <UiIcon :icon class="left-icon" />
+    <span :class="{ required }" class="typo c2-semi-bold label"><slot /></span>
+    <a v-if="href" :href class="link">
+      <span class="typo p3-regular-underline">{{ $t('learn-more') }}</span>
+      <UiIcon :icon="faUpRightFromSquare" class="link-icon" />
+    </a>
+  </div>
+</template>
+
+<script lang="ts" setup>
+import UiIcon from '@core/components/icon/UiIcon.vue'
+import type { IconDefinition } from '@fortawesome/fontawesome-common-types'
+import { faUpRightFromSquare } from '@fortawesome/free-solid-svg-icons'
+
+defineProps<{
+  status: 'normal' | 'warning' | 'danger'
```

Comment:
Why not create a dedicated type for this, since it seems all inputs component have this `status` prop?
```ts
type InputStatus = 'normal' | 'warning' | 'danger'
```

If more components use these statuses, maybe create a more generic type?